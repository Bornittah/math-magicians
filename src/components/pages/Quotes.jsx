import React from 'react';

const Quotes = () => (
  <main>
    <div className="qoute flex align-center justify-center">
      <p>
        Mathematics is not about numbers, equations, computations or
        algorithms; it&apos;s about understanding. - William Paul Thurston
      </p>
    </div>
  </main>
);

export default Quotes;
