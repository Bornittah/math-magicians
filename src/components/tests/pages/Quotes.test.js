import renderer from 'react-test-renderer';
import Quotes from '../../pages/Quotes';

describe('Testing Quote component', () => {
  it('Testing with snapshot', () => {
    const tree = renderer.create(<Quotes />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
