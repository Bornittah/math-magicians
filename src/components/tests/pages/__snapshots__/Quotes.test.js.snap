// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Quote component Testing with snapshot 1`] = `
<main>
  <div
    className="qoute flex align-center justify-center"
  >
    <p>
      Mathematics is not about numbers, equations, computations or algorithms; it's about understanding. - William Paul Thurston
    </p>
  </div>
</main>
`;
